### PR TITLE
Updated the migrating.md document

### PR DESCRIPTION
**Pull request by KuJoe**
- New guidance to update the mastodon database user password
- Instructions to copy over the Redis database
- Added new flag to improve pg_restore performance
- A suggestion to copy the SSL certificates over
- Added the commands needed to complete the migration along with an optional command for Elasticsearch.

### CODE REVIEW

File: content/en/admin/migrating.md [hunk 1]
```diff
@@ -76,10 +88,16 @@ This will probably take some time, and you’ll want to avoid re-copying unneces
 rsync -avz ~/live/public/system/ mastodon@example.com:~/live/public/system/
 ```
 
-You’ll want to re-run this if any of the files on the old server change.
+You’ll want to re-run this if any of the files on the old server change.  
 
 You should also copy over the `.env.production` file, which contains secrets.
 
+Now copy your redis database over (adjust the location of your redis database as needed). On your old machine, as the `root` user, run:
```
Comment (vmstan) [inline content/en/admin/migrating.md:95]:
```suggestion
Now copy your Redis database over (adjust the location of your Redis database as needed). On your old machine, as the `root` user, run:
```

File: content/en/admin/migrating.md [hunk 2]
```diff
@@ -15,17 +15,19 @@ This guide was written with Ubuntu Server in mind; your mileage may vary for oth
...
-1. Set up a new Mastodon server using the [Production Guide]({{< relref "install" >}}) (however, don’t run `mastodon:setup`).
-2. Stop Mastodon on the old server (e.g. `systemctl stop 'mastodon-*.service'`).
+1. Set up a new Mastodon server using the [Production Guide]({{< relref "install" >}}) (however, don’t run `mastodon:setup` and only leave the PostgreSQL service running).
+2. Stop Mastodon and Redis on the old server (e.g. `systemctl stop 'mastodon-*.service'` and `systemctl stop redis-server.service).
 3. Dump and load the PostgreSQL database using the instructions below.
 4. Copy the `system/` files using the instructions below. (Note: if you’re using S3, you can skip this step.)
 5. Copy the `.env.production` file.
-6. Run `RAILS_ENV=production bundle exec rails assets:precompile` to compile Mastodon
-7. Run `RAILS_ENV=production ./bin/tootctl feeds build` to rebuild the home timelines for each user.
+6. Copy the Redis database from `/var/lib/redis/` to the new server.
```
Comment (vmstan) [inline content/en/admin/migrating.md:23]: Is this the right way to go about this, or should they be running an actual backup of the database?

File: content/en/admin/migrating.md [hunk 3]
```diff
@@ -15,17 +15,19 @@ This guide was written with Ubuntu Server in mind; your mileage may vary for oth
...
+7. Run `RAILS_ENV=production bundle exec rails assets:precompile` to compile Mastodon
 8. Start Mastodon on the new server.
-9. Update your DNS settings to point to the new server.
-10. Update or copy your Nginx configuration, and re-run LetsEncrypt as necessary.
-11. Enjoy your new server!
+9. Run `RAILS_ENV=production ./bin/tootctl feeds build` to rebuild the home timelines for each user.
```
Comment (vmstan) [inline content/en/admin/migrating.md:26]: Redis needs to be up for this.

File: content/en/admin/migrating.md (hunk 2 above)
Comment (KuJoe) [reply content/en/admin/migrating.md:23]: I was under the impression that the Redis server is stopped it writes everything to the dump.rdb file allowing it to just be copied over, but I could be wrong. I will research this more.

File: content/en/admin/migrating.md (hunk 3 above)
Comment (KuJoe) [reply content/en/admin/migrating.md:26]: Correct, I had that information in the "detailed steps" but forgot to include that in the "basic steps". I also put the wrong order for starting the services which I'm fixing now.

File: content/en/admin/migrating.md (hunk 2 above)
Comment (KuJoe) [reply content/en/admin/migrating.md]: I added the commands to save the Redis database via the redis-cli just to be extra safe. As long as the Sidekiq services are stopped that should be sufficient.

**Review by andypiper — APPROVED** [review]

**Comment by vmstan** [comment]
@andypiper are these good to merge?

**Pull request merged**